unixfs/mod/dagmodifier.go

```diff
@@ -0,0 +1,428 @@
+package mod
```

Comment:
did the dag modifier change at all? i cant tell. it would be useful to put this in its own pr and in two separate commits.